src/app/pages/ValidatorDetailsPage/index.tsx

```diff
@@ -14,25 +17,35 @@ import { ValidatorCommission } from '../../components/Validators/ValidatorCommis
 import { ValidatorCumulativeVoting } from '../../components/Validators/ValidatorCumulativeVoting'
 import { ValidatorTitleCard } from './ValidatorTitleCard'
 import { useRequiredScopeParam } from 'app/hooks/useScopeParam'
-import { useLoaderData } from 'react-router-dom'
 import { AddressLoaderData } from 'app/utils/route-utils'
 import { ValidatorSnapshot } from './ValidatorSnapshot'
 
+export type ValidatorDetailsContext = {
+  scope: SearchScope
+  address: string
+}
+
+export const useValidatorDetailsProps = () => useOutletContext<ValidatorDetailsContext>()
```

Comment:
This probably still breaks hotreload (but i didn't check. entityId API annoying)

Reply:
ah true, fixed